04_numeronym/sksandani/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	inputString      []string
+	numeronymsOutput []string
+}{
+	{[]string{}, []string{}},
+	{[]string{" 12345", "+12345", "-1234"}, []string{"135", "+45", "-34"}},
+	{[]string{"accessibility", "Kubernetes", "abc"}, []string{"a11y", "K8s", "abc"}}}
```

Comment:
Would like to see a test involving an empty string `""`. If you want to support unicode you will have to change your `getNumeronyms` function to avoid using `bytes.Buffer`. Ask @juliaogris about this.
Also, put the final closing `}` on a new line.